### PR TITLE
SC-104 Remote openXDA Instances pages

### PR DESCRIPTION
**Pull request by KulinduC**
Replaced window.innerHeights with flex

### CODE REVIEW

File: Source/Applications/SystemCenter/wwwroot/Scripts/TSX/SystemCenter/RemoteXDA/RemoteXDAInstance.tsx [hunk 1]
```diff
@@ -103,16 +103,10 @@ function RemoteXDAInstance(props: IProps) {
             <hr />
```
Comment (gcsantos-gpa) [inline Source/Applications/SystemCenter/wwwroot/Scripts/TSX/SystemCenter/RemoteXDA/RemoteXDAInstance.tsx:103]: Seems to no longer draw a line. Something about changing the parent styling likely lead to this. Looking at the element, it looks like it got it's width overwritten. Setting the width to 100% in the styling seems to fix it.

File: Source/Applications/SystemCenter/wwwroot/Scripts/TSX/SystemCenter/RemoteXDA/SystemSettingsTab.tsx [hunk 2]
```diff
@@ -68,15 +68,15 @@ const SystemSettingsTab = (props: IProps) => {
...
+        <div className="card" style={{ flex: 1, display: 'flex', flexDirection: 'column', overflow: 'hidden' }}>
             <div className="card-header">
                 <div className="row">
                     <div className="col">
                         <h4>Remote openXDA Instance Connection Information:</h4>
                     </div>
                 </div>
             </div>
-            <div className="card-body" style={{ maxHeight: window.innerHeight - 315, overflowY: 'auto' }}>
+            <div className="card-body" style={{ flex: 1, overflowY: 'auto' }}>
```
Comment (gcsantos-gpa) [inline Source/Applications/SystemCenter/wwwroot/Scripts/TSX/SystemCenter/RemoteXDA/SystemSettingsTab.tsx:79]: overflow here should be hidden. Overflowing should be handled by child table.

File: Source/Applications/SystemCenter/wwwroot/Scripts/TSX/SystemCenter/RemoteXDA/RemoteMeterTab.tsx [hunk 3]
```diff
@@ -147,8 +147,9 @@ const RemoteMeterTab = (props: IProps) => {
                         setSortKey(d.colField);
                     }
                 }}
+                TableStyle={{ padding: 0, width: '100%', tableLayout: 'fixed', display: 'flex', flexDirection: 'column', overflow: 'hidden' }}
```
Comment (gcsantos-gpa) [inline Source/Applications/SystemCenter/wwwroot/Scripts/TSX/SystemCenter/RemoteXDA/RemoteMeterTab.tsx:150]: Overflow for the table should be scroll.

**Review by gcsantos-gpa — CHANGES_REQUESTED** [review]

File: Source/Applications/SystemCenter/wwwroot/Scripts/TSX/SystemCenter/RemoteXDA/SystemSettingsTab.tsx (hunk 2 above)
Comment (KulinduC) [reply Source/Applications/SystemCenter/wwwroot/Scripts/TSX/SystemCenter/RemoteXDA/SystemSettingsTab.tsx:79]: There is no child table for this tab

**Review by gcsantos-gpa — APPROVED** [review]

**Pull request merged**